docs/t-sql/database-console-commands/dbcc-traceon-trace-flags-transact-sql.md

```diff
@@ -234,6 +234,7 @@ Trace flag behavior may not be supported in future releases of [!INCLUDE[ssNoVer
 | <a id="tf9591"></a>**9591** | Disables log block compression in Always On Availability Groups. Log block compression is the default behavior used with both synchronous and asynchronous replicas in [!INCLUDE[ssSQL11](../../includes/sssql11-md.md)] and [!INCLUDE[ssSQL14](../../includes/sssql14-md.md)]. In [!INCLUDE[sssql16-md](../../includes/sssql16-md.md)], compression is only used with asynchronous replica.<br /><br />**Scope:** Global or session. |
 | <a id="tf9592"></a>**9592** | Enables log stream compression for synchronous availability groups. This feature is disabled by default on synchronous availability groups because compression adds latency. For more information, see [Tune compression for availability group](../../database-engine/availability-groups/windows/tune-compression-for-availability-group.md).<br /><br />**Scope:** Global or session. |
 | <a id="tf9708"></a>**9708** | Enables collection of event publishing metrics for extended event sessions. For more information, see [sys.dm_xe_session_events](../../relational-databases/system-dynamic-management-views/sys-dm-xe-session-events-transact-sql.md).<br /><br />**Applies to:** [!INCLUDE[sssql22-md](../../includes/sssql22-md.md)] and later versions.<br /><br />**Scope:** Global only. |
+| <a id="tf9810"></a>**9810** | Disables In-Memory OLTP enging from reclaiming Thread Local Storage (TLS) memory. Not reclaiming TLS memory is the default behavior in SQL Server 2019 and earlier version. In SQL Server 2022, a new memory optimization was introduced which causes Hekaton engine to reclaim TLS memory and to reduce possibily of out-of-memory issues. This trace flag reverts to the behavior prior to SQL Server 2022. For more information, see .<br /><br />**Applies to:** [!INCLUDE[sssql22-md](../../includes/sssql22-md.md)] and later versions.<br /><br />**Scope:** Global only. |
```

Comment:
Workshopping the main content. I see there's a "more info" link still missing, but I hope this helps in the meantime.

```suggestion
| <a id="tf9810"></a>**9810** | Disables the In-Memory OLTP engine from reclaiming Thread Local Storage (TLS) memory. In [!INCLUDE[sssql19-md](../../includes/sssql19-md.md)] and earlier versions, not reclaiming TLS memory is the default behavior. In [!INCLUDE[sssql22-md](../../includes/sssql22-md.md)], a new memory optimization was introduced that causes the In-Memory OLTP engine to reclaim TLS memory, and to reduce the possibility of out-of-memory issues. This Trace Flag reverts to the behavior prior to [!INCLUDE[sssql22-md](../../includes/sssql22-md.md)]. For more information, see .<br /><br />**Applies to:** [!INCLUDE[sssql22-md](../../includes/sssql22-md.md)] and later versions.<br /><br />**Scope:** Global only. |
```